client/data/plugins/use-monitor-settings-mutation.ts

```diff
@@ -47,15 +48,38 @@ export const useMonitorSettingsQuery = (
 };
 
 export function useCreateMonitorSettingsMutation( siteSlug: SiteSlug, queryOptions = {} ) {
+	const MAX_RETRIES = 3;
+	const isMonitorNotActiveErrorMsg = 'Monitor is not active.';
+
+	const isMonitorNotActive = ( data: UpdateMonitorSettingsCreate ) => {
+		return data?.settings && data?.settings?.monitor_active === false;
+	};
+
+	const isMonitorNotActiveError = ( error: Error ) => {
+		return error.message === isMonitorNotActiveErrorMsg;
+	};
+
 	const mutation = useMutation( {
-		mutationFn: ( params: object ) =>
-			wpcomRequest( {
+		mutationFn: async ( params: object ) => {
+			const response: UpdateMonitorSettingsCreate = await wpcomRequest( {
 				path: `/sites/${ siteSlug }/jetpack-monitor-settings`,
 				apiNamespace: 'wpcom/v2',
 				method: 'POST',
 				body: params,
-			} ),
+			} );
+			if ( isMonitorNotActive( response ) ) {
+				throw new Error( isMonitorNotActiveErrorMsg );
+			}
+			return response;
+		},
 		...queryOptions,
+		retry: ( failureCount, error ) => {
+			if ( isMonitorNotActiveError( error ) && failureCount < MAX_RETRIES ) {
+				return true;
```

Comment:
Yep, I added a new event to track here `calypso_scheduled_updates_retry_monitor_settings`. I've also added a 3-second delay to the initial request after the monitor module is activated. Hopefully, this will address most of the issues and no retries are needed. But let's see how goes with the event.